nixos/modules/services/web-servers/garage.nix

```
@@ -49,7 +49,7 @@ in
 
           replication_mode = mkOption {
             default = "none";
-            type = types.enum ([ "none" "1" "2" "3" 1 2 3 ]);
+            type = types.enum ([ "none" "1" "2" "3" "2-dangerous" "3-dangerous" 1 2 3 ]);
```

While we are at it, is there any particular reason why `3-degraded` is missing from this `enum`? :)
Docs: https://garagehq.deuxfleurs.fr/documentation/reference-manual/configuration/#replication-mode

```suggestion
            type = types.enum ([ "none" "1" "2" "3" "2-dangerous" "3-dangerous" "3-degraded" 1 2 3 ]);
```

---

Good shout!